dummy/main.py

```diff
@@ -190,26 +192,31 @@ def _register_entities(self):
                         max_temp=70.0)
                 logger.info('Registered %s' % unit)
                 self._hot_water = unit
+                self._hot_water_dummy = HotWaterDummy(unit, report_status=self.report_hot_water_status)
```

Comment:
maybe only create a hot_water unit if it enabled through the config, same as with the sensors.

Reply:
that is the case, a few lines above there is the 
```if self._config.get('hot_water', False):```

Reply:
I did see an issue with updated config though, the existing unit needs to be stopped. I've added that.